torch/distributed/fsdp/flat_param.py

```diff
@@ -2100,6 +2286,46 @@ def _safe_setattr_tensor_or_param(
     setattr(module, param_name, tensor_or_param)
 
 
+def _convert_to_params(
+    tensors: List[Union[torch.Tensor, nn.Parameter]]
+) -> List[nn.Parameter]:
+    return [t if isinstance(t, nn.Parameter) else nn.Parameter(t) for t in tensors]
+
+
+def _detach_if_needed(param_or_tensor: Union[nn.Parameter, Tensor]) -> Tensor:
```

Comment:
nit: would this be better in a utils file?

Reply:
Yes, we can move it to a utils file.